package.json

```diff
@@ -13,7 +13,7 @@
     "appveyor:test": "npm run test",
     "build": "cross-env NODE_ENV=production babel src -d dist --ignore 'src/**/*.test.js'",
     "clean": "del-cli dist",
-    "lint": "eslint --cache src test",
+    "lint": "eslint --cache src test --ignore-pattern fixtures",
```

Comment:
Could this be declared in the `eslintrc.js` instead ?

Reply:
@michael-ciniawsky let's discuss about this in `webpack-defaults`

Reply:
https://github.com/webpack-contrib/webpack-defaults/pull/130